app/src/main/java/com/group/a/gradeapp/MainActivity.java

```diff
@@ -2,13 +2,73 @@
 
 import androidx.appcompat.app.AppCompatActivity;
 
+import android.content.Intent;
 import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
 
 public class MainActivity extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        Button create_account_button = findViewById(R.id.create_account);
+        create_account_button.setOnClickListener(new View.OnClickListener(){
+            @Override
+            public void onClick(View v) {
+                // call the create account activity
+                Log.d("MainActivity", "onClick for create account called");
+                Intent intent = new Intent(MainActivity.this, CreateAccountActivity.class);
+                startActivity(intent);
+
+            }
+        });
+
+
+        Button login_button = findViewById(R.id.login);
+        login_button.setOnClickListener(new View.OnClickListener(){
+            @Override
+            public void onClick(View v) {
+                // call the login Activity
+                Log.d("Login", "onClick for login activity called");
+                Intent intent = new Intent(MainActivity.this, LoginActivity.class);
+                startActivity(intent);
+
+            }
```

Comment:
Great work this week! The MainActivity.java was easy to understand and the good habit of adding comments.